applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml

```diff
@@ -96,5 +96,17 @@ template:
     entity_check: "all"
     filepath: {{{ openshift_filtered_path(default_api_path, default_jqfilter) }}}
     yamlpath: "[:]"
-    xccdf_variable: var_apiserver_encryption_type
-    regex_data: true
+    check_existence: "at_least_one_exists"
+    values:
```

Comment:
Hmm, besides the error regarding the name:
```xml
<ind:yamlfilecontent_state id="oval:ssg-state_api_server_encryption_provider_cipher:ste:1" version="1">
  <ind:value datatype="record" entity_check="all">
    <oval-def:field name="#" datatype="string" operation="equals">aescbc</oval-def:field>
    <oval-def:field name="#" datatype="string" operation="equals">aesgcm</oval-def:field>
  </ind:value>
</ind:yamlfilecontent_state>
```
`OpenSCAP Error: File '/__w/content/content/build/ssg-ocp4-ds.xml' line 31703: Element '{http://oval.mitre.org/XMLSchema/oval-definitions-5}field': Duplicate key-sequence ['#'] in unique identity-constraint '{http://oval.mitre.org/XMLSchema/oval-definitions-5#independent}UniqueYamlFileValueFieldName'.`

I had not realized that this is checking for an entry with both values, not for an entry with either of the values.
Was there any problem with the regex that was being used before?